iceberg/iceberg-handler/src/test/queries/positive/update_iceberg_unpartitioned_parquet.q

```diff
@@ -0,0 +1,26 @@
+drop table if exists tbl_ice;
+create external table tbl_ice(a int, b string, c int) stored by iceberg tblproperties ('format-version'='2');
+
+-- update using simple predicates
+insert into tbl_ice values (1, 'one', 50), (2, 'two', 51), (3, 'three', 52), (4, 'four', 53), (5, 'five', 54), (111, 'one', 55), (333, 'two', 56);
+update tbl_ice set b='Changed' where b in ('one', 'four') or a = 22;
```

Comment:
It may be good to cover all datatypes supported as well (just guardrails so that it doesn't break later?).

Reply:
Added unit tests for that.
Thanks for the idea, actually found some issues when the tests were run!